docs/articles/infrastructure/ci-cd/jenkins.rst

```diff
@@ -70,4 +70,43 @@ Normally we use the proposed plugins during installation and then add the follow
 - Copy Artifact Plugin
 - Build Monitor View
 
+Naming Jenkings jobs
+--------------------
+
+This part describes how to name Jenkins jobs in the Software Factory in a
+consistent manner. Intended reader is anyone that needs to add a Jenkins job.
+
+Syntax, using EBNF (Extended Backus-Naur Form @ Wikipedia) notation:
+    * job_name = component SEP job_type SEP build_interval
+    * build_interval = "nightly" | "hourly" | "weekly" | "on commit" | "manual"
+    * job_type = build_type | test_type | maintainence_type | composite_build_type
+    * test_type = variant_name SEP ( % Widely acepted test-type name, such as
+      "integtation test", "stress test", etc % )
+    * build_type = variant_name SEP ("release" | "debug")
+    * maintenance_type = "backup" | "cleanup"
+    * SEP = "-"
+    * component =  yocto_image_name SEP yocto_artifact_type | component_name
+    * yocto_image_name = % This is a valid image name in a Yocto build %
+    * yocto_artifact_type = "IMAGE" | "SDK"
+    * variant_name = yocto_variant | cpu_arch
+    * component_name = ([A-Z] [a-z] [0-9] [special_chars])+ % This should be a
+      "logical" component name, which has a real-world meaning in the project %
+    * special_chars = "_"
+    * cpu_arch = % Valid architecture as defined by STANDARDS_ORG / GNU arch
+      triples, for example: aarch64 | x86_64 | x86 %
+    * yocto_variant = % Currently, we have these: rpi3_nogfx | qemu-x86-64_nogfx
```

Comment:
Because of things like this I'm not sure this article should go in into the blueprint.